site/realm/data_sources/mongodb-atlas/aiidprod/reports/rules.json

```diff
@@ -32,19 +32,35 @@
             "search": true,
             "additional_fields": {}
         },
+        {
+            "name": "Variant",
```

Comment:
I think this rule allows any report with `text_inputs` and `text_outputs` fields set to be created/deleted by anonymous users.

Looks like a security issue 🤔

Reply:
I just added another condition to this rule
```
"title": {
    "$eq": ""
}
```
to only match Variants.
The idea is to allow anonymous users to create Variants